### ISSUE TITLE
Movers with from_dp

### ISSUE THREAD

**Issue by Mathadon** [lbl-srg/modelica-buildings#474]
Pressure drop components have a parameter `from_dp` that allow to invert the pressure drop equation. Using this parameter can greatly simplify non-linear algebraic loops after tearing.

It would be nice to have the same parameter for movers since this should lead to similar improvements when connecting multiple pumps in parallel.

**Comment by mwetter** [comment]
This would be very hard, e.g., a multiple week job and makes the model more complicated. The reason is because `_dpMachine` (in `FlowMachineInterface`) is computed carefully to satisfy the conditions in http://simulationresearch.lbl.gov/wetter/download/2013-IBPSA-Wetter.pdf

If you were to compute `VMachine_flow` from `_dpMachine`, then you would have to do a similar analysis to avoid numerical problems.

However, if your pumps are all the same, we could add a parameter `nParallel` and then use `VMachine_flow/nParallel` as n parallel pumps produce n times the flow rate at the same head. The MSL has a similar implementation. We would also need to adjust accordingly the power consumption.

**Comment by Mathadon** [comment]
I agree that it would not be easy. Having nParallel would not solve my problem.

I also do not have time to look into this in detail so I guess it will not happen any time soon, but we can leave this issue open in case anyone else wants to have a go at it?
